bundles/admin/admin-layereditor/view/AdminLayerForm/AdminLayerFormHandler.js

```diff
@@ -14,7 +14,8 @@ const __VALIDATOR_CACHE = {};
 class UIHandler extends StateHandler {
     constructor (consumer) {
         super();
-        const mapmodule = Oskari.getSandbox().findRegisteredModuleInstance('MainMapModule');
+        this.sandbox = Oskari.getSandbox();
+        this.mapmodule = Oskari.getSandbox().findRegisteredModuleInstance('MainMapModule');
```

Comment:
It doesn't look like you would need to store a reference to mapmodule in `this` as it's not used by any of these changes?

Reply:
Changed back to `const mapmodule ...`